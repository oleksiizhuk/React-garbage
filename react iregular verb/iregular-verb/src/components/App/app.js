import React, {Fragment, Component} from 'react';

export default class App extends Component{

    render() {
        return (
            <div></div>
        )
    }
}
