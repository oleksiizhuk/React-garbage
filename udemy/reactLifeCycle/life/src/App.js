import React, { Component } from 'react';
import { Switch, Router, Route } from 'react-router-dom';
import { createBrowserHistory } from 'history';
import Page from './components/layout/Page/Page';
import Home from './components/pages/Home/Home';
import About from './components/pages/About/About';
import './scss/index.scss';

export default class App extends Component {
  constructor(props) {
    super(props);
    this.history = createBrowserHistory();
  }

  render() {
    return (
      <Router history={this.history}>
        <Page>
          <Switch>
            <Route exact strict path="/" component={Home} />
            <Route path="/about/" component={About} />
          </Switch>
        </Page>
      </Router>
    );
  }
}
