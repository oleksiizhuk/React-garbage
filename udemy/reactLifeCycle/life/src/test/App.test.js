import React from 'react';
import TestRenderer from 'react-test-renderer';

import App from '../App';

it('renders correctly when there are no items', () => {
  const tree = TestRenderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
