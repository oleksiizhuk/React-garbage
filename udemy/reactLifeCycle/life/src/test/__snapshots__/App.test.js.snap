// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
Array [
  <h2>
    Header
  </h2>,
  <div>
    test
  </div>,
  <footer
    className="footer"
  >
    <div
      className="container"
    >
      <span>
        Â© Copyright 2019 by Oleksii Zhukk
      </span>
    </div>
  </footer>,
]
`;
